src/Converter.cpp

```diff
@@ -141,21 +152,59 @@ int convertStreams(
 int convertSamples(Converter& conv, pocolog_cpp::InputDataStream* stream,
     const int start, const int end, const int verbose)
 {
+    
+    float next_report_progress = 0.0;
```

Comment:
To keep a consistent style throughout this repository, please also use camel case, not snake case.